src/components/10-atoms/input-text/index.js

```diff
@@ -267,10 +266,14 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
         });
       }
 
-      if (hasAtLeastOneDigit && dotsCounted <= 1) {
-        this.invalid = false;
+      if (hasAtLeastOneDigit) {
         const valueDecimalsOnly = value.replace(/[^0-9.]/g, '');
-        return this.currencyFormatter.format(valueDecimalsOnly);
+
+        // eslint-disable-next-line no-restricted-globals
+        if (!isNaN(valueDecimalsOnly)) {
+          this.invalid = false;
+          return this.currencyFormatter.format(valueDecimalsOnly);
+        }
```

Comment:
Would use parseFloat(valueDecimalsOnly) as @markus-walther recommended. It is cleaner and more speaking.

Reply:
`if (hasAtLeastOneDigit) {
        const valueDecimalsOnly = value.replace(/[^0-9.]/g, '');
        return this.currencyFormatter.format(parseFloat(valueDecimalsOnly));
 }`